Frameworks/UIKit.Xaml/Label.xaml.cpp

```diff
@@ -53,12 +53,45 @@ Canvas^ Label::SublayerCanvas::get() {
     if (!_sublayerCanvas) {
         _sublayerCanvas = ref new Canvas();
         _sublayerCanvas->Name = "Sublayers";
-        Children->Append(_sublayerCanvas);
+
+        // If we have a border make sure the sublayer canvas is inserted before it
```

Comment:
is this to make sure that border's zindex is bigger than sublayer thus display on top?